sites/partners/src/components/settings/PreferenceDrawer.tsx

```diff
@@ -94,9 +94,14 @@ const PreferenceDrawer = ({
 
   const optOutQuestion = watch("canYouOptOutQuestion")
 
+  const isAdditionalDetailsEnabled = profile?.jurisdictions?.some(
```

Comment:
Question/Comment/Thought: Do you mind expanding on using "some" here? I remember hearing that for release we'd want to hide or show for all jurisdictions but surprised to see that we wouldn't still use the jurisdiction of the preference if we're putting it on the jurisdiction table as opposed to env. Seems like we could leave that option open code-wise while updating the db as expected. What are your thoughts?

Reply:
Discussed offline.

The issue with this page is that the jurisdiction of the preference is not known until the user selects it from the dropdown. And that can be done before or after the rest of the fields are filled out. So we can't rely on it

Also, we are planning on only releasing this to all jurisdictions at once so the "some" shouldn't be a problem. However, it is something to note for future expansions. We could potentially remove the collect geocoding information if they select a jurisdiction that doesn't have it.